common/test/acceptance/pages/studio/overview.py

```diff
@@ -1037,19 +1027,48 @@ def policy(self, grading_label):
         ).fulfill()
 
     @property
+    def is_staff_lock_visible(self):
+        """
+        Returns true if the staff lock option is visible, either as a checkbox
+        (section and unit levels) or a radio button (subsection level).
+        """
+        return self.find_css('#staff_lock').visible or self.find_css('input[name=content-visibility]').visible
+
+    def ensure_staff_lock_visible(self):
+        """
+        Ensures the staff lock option is visible, clicking on the advanced tab
+        if needed.
+        """
+        if not self.is_staff_lock_visible:
+            self.find_css(".settings-tab-button[data-tab=advanced]").click()
+
+    @property
     def is_explicitly_locked(self):
         """
         Returns true if the explict staff lock checkbox is checked, false otherwise.
         """
-        return self.find_css('#staff_lock')[0].is_selected()
+        self.ensure_staff_lock_visible()
+        if self.find_css('input[name=content-visibility]').visible:
+            return self.find_css('input[name=content-visibility][value=staff_only]')[0].is_selected()
+        else:
+            return self.find_css('#staff_lock')[0].is_selected()
 
     @is_explicitly_locked.setter
     def is_explicitly_locked(self, value):
```

Comment:
I mean, it's a property setter, so when called as `modal.is_explicitly_locked = True`, it seems to indicate what's happening pretty clearly IMO. Do you have a suggestion for a better name?